src/app/dashboard/dashboard.component.ts

```diff
@@ -9,14 +15,17 @@ import { Observable } from 'rxjs';
 })
 export class DashboardComponent implements OnInit {
   item$: Observable<any>;
+  filterObj: Filters;
 
   constructor(private afs: AngularFirestore) { }
 
   ngOnInit() {
-    this.item$ = this.afs.collection('pack-items', ref => ref.where('category', '==', 'Clothing')).valueChanges();
+    // this.item$ = this.afs.collection('pack-items', ref => ref.where('category', '==', 'Clothing')).valueChanges();
   }
 
-  handleFilter(filterObj) {
-    console.log(filterObj);
+  handleFilter(filter) {
+    console.log(filter);
+    this.filterObj = Object.assign({}, this.filterObj, filter);
```

Comment:
We can use object spreading: `this.filterObj = { ...this.filterObj, ...filter }`